### PR TITLE
release v1.1.0-beta.2

### LINKED ISSUE #1760: release tidb-operator v1.1.0-beta.2
This is a track issue for tidb-operator v1.1.0-beta.2.

previous: 
- https://github.com/pingcap/tidb-operator/releases/tag/v1.1.0-beta.1
- https://github.com/pingcap/tidb-operator/pull/1459

pending PRs:

- [x] https://github.com/pingcap/tidb-operator/pull/1759
- [x] https://github.com/pingcap/tidb-operator/pull/1762
- [x] https://github.com/pingcap/tidb-operator/pull/1763
- [x] https://github.com/pingcap/tidb-operator/pull/1761 (cherry-pick of  https://github.com/pingcap/tidb-operator/pull/1640)

### PR DESCRIPTION
**Pull request by cofyc**
### What problem does this PR solve? 

fixes #1760 

release notes are generated via the following command:

```
go get k8s.io/release/cmd/release-notes@v0.2.0
release-notes --start-rev v1.1.0-beta.1 --end-rev release-1.1 --branch release-1.1 --github-org pingcap --github-repo tidb-operator --output note.md --debug --format markdown -requiredAuthor ''
```

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.1.md [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+
+
 # TiDB Operator v1.1.0-beta.1 Release Notes
 
 This is a pre-release of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
 
 ## Changes since v1.0.0
 
-### Action required
+### Action Required
```
Comment (cofyc) [inline CHANGELOG-1.1.md]: titles generated by `release-notes` are capitalized. I update this then we don't need to modify the output of `release-notes`.

File: CHANGELOG-1.1.md [hunk 2]
```diff
@@ -17,7 +46,7 @@ This is a pre-release of `v1.1.0`, which focuses on the usability, extensibility
   
   All images' time zone maintained by `tidb-operator` is `UTC`. If you use your own images, you need to make sure that the time zone inside your images is `UTC`. 
 
-### Other notable changes
+### Other Notable Changes
```
Comment (cofyc) [inline CHANGELOG-1.1.md]: ditto

File: CHANGELOG-1.1.md [hunk 3]
```diff
@@ -1,10 +1,39 @@
...
+
+## Other Notable Changes
+
+- Allow users to configure affinity and tolerations for `Backup` and `Restore` ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
+- Make AdvancedStatefulSet and Admission Webhook could work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
+- Add a basic deployment example of TiDB cluster ([#1573](https://github.com/pingcap/tidb-operator/pull/1573), [@aylei](https://github.com/aylei))
+- Support TidbCluster Auto-scaling feature based by CPU average utilization load. ([#1731](https://github.com/pingcap/tidb-operator/pull/1731), [@Yisaer](https://github.com/Yisaer))
+- Support user-defined tidb server/client certificate ([#1714](https://github.com/pingcap/tidb-operator/pull/1714), [@weekface](https://github.com/weekface))
+- Add an option for tidb-backup chart to allow reusing existing PVC or not for restore ([#1712](https://github.com/pingcap/tidb-operator/pull/1712), [@sre-bot](https://github.com/sre-bot))
```
Comment (Yisaer) [inline CHANGELOG-1.1.md]: I think we should refine the log by changing the `sre-bot` author.

Comment (cofyc) [reply CHANGELOG-1.1.md]: ah, I'll fix this.

File: CHANGELOG-1.1.md [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
+
+## Action Required
+
+- `--default-storage-class-name` and `--default-backup-storage-class-name `are abandoned, storage class defaults to Kubernetes default storage class right now. If you have set default storage class different than Kubernetes default storage class, please set them explicitly in your tidb cluster helm or YAML files. ([#1581](https://github.com/pingcap/tidb-operator/pull/1581), [@cofyc](https://github.com/cofyc))
+
+## Other Notable Changes
+
+- Allow users to configure affinity and tolerations for `Backup` and `Restore` ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
+- Make AdvancedStatefulSet and Admission Webhook could work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
+- Add a basic deployment example of TiDB cluster ([#1573](https://github.com/pingcap/tidb-operator/pull/1573), [@aylei](https://github.com/aylei))
```
Comment (tennix) [inline CHANGELOG-1.1.md]: This needs to be clarified a bit. It's for the new CRD based cluster management.

File: CHANGELOG-1.1.md [hunk 5]
```diff
@@ -1,10 +1,39 @@
...
+- Support TidbCluster Auto-scaling feature based by CPU average utilization load. ([#1731](https://github.com/pingcap/tidb-operator/pull/1731), [@Yisaer](https://github.com/Yisaer))
```
Comment (tennix) [inline CHANGELOG-1.1.md]: s/based by/based on/?

File: CHANGELOG-1.1.md (hunk 3 above)
Comment (tennix) [reply CHANGELOG-1.1.md]: Yes, with the generated release note, we still should refine some words and sentences. The PR and author should be the original PR and author, other than sre-bot.

**Comment by Yisaer** [comment]
I think we should add the document to introduce the way to manage tidbcluster with simple yaml files including `TidbCluster`/ `TidbMonitor`/`TidbInitializer`/`TidbClusterAutoScaler`.

File: CHANGELOG-1.1.md (hunk 4 above)
Comment (cofyc) [reply CHANGELOG-1.1.md]: cc @aylei can you update the release note in your PR #1573? I'll regenerate this changelog later.

File: CHANGELOG-1.1.md (hunk 3 above)
Comment (cofyc) [reply CHANGELOG-1.1.md]: I'll figure out how to filter out `sre-bot`.

Comment (cofyc) [reply CHANGELOG-1.1.md]: fixed

File: CHANGELOG-1.1.md (hunk 4 above)
Comment (cofyc) [reply CHANGELOG-1.1.md]: updated

File: CHANGELOG-1.1.md (hunk 5 above)
Comment (cofyc) [reply CHANGELOG-1.1.md]: updated

File: CHANGELOG-1.1.md [hunk 6]
```diff
@@ -1,10 +1,40 @@
...
+- Add a basic deployment example of managing TiDB cluster with custom resources only. ([#1573](https://github.com/pingcap/tidb-operator/pull/1573), [@aylei](https://github.com/aylei))
+- Support TidbCluster Auto-scaling feature based on CPU average utilization load. ([#1731](https://github.com/pingcap/tidb-operator/pull/1731), [@Yisaer](https://github.com/Yisaer))
+- Support user-defined tidb server/client certificate ([#1714](https://github.com/pingcap/tidb-operator/pull/1714), [@weekface](https://github.com/weekface))
+- Add an option for tidb-backup chart to allow reusing existing PVC or not for restore ([#1708](https://github.com/pingcap/tidb-operator/pull/1708), [@mightyguava](https://github.com/mightyguava))
+- Add `resources`, `imagePullPolicy` and `nodeSelector` field for tidb-backup chart ([#1705](https://github.com/pingcap/tidb-operator/pull/1705), [@mightyguava](https://github.com/mightyguava))
+- Add more SANs to tidb server certificate ([#1702](https://github.com/pingcap/tidb-operator/pull/1702), [@weekface](https://github.com/weekface))
```
Comment (weekface) [inline CHANGELOG-1.1.md]:
```suggestion
- Add more SAN(Subject Alternative Name)s to tidb server certificate ([#1702](https://github.com/pingcap/tidb-operator/pull/1702), [@weekface](https://github.com/weekface))
```

**Comment by cofyc** [comment]
> I think we should add the document to introduce the way to manage tidbcluster with simple yaml files including `TidbCluster`/ `TidbMonitor`/`TidbInitializer`/`TidbClusterAutoScaler`.

sure, we need add some basic guide in this repo and more detail docs in docs/docs-cn repos when 1.1.0 is GA.

**Comment by cofyc** [comment]
@Yisaer @weekface @tennix PTAL again.

**Comment by cofyc** [comment]
hi, @yikeke do you have time to review our release notes in this PR?

**Comment by yikeke** [comment]
> hi, @yikeke do you have time to review our release notes in this PR?

Okay.

File: CHANGELOG-1.1.md [hunk 7]
```diff
@@ -1,10 +1,40 @@
+# TiDB Operator v1.1.0-beta.2 Release Notes
+
+This is a pre-release of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
+
+## Changes since v1.1.0-beta.1
+
+## Action Required
+
+- `--default-storage-class-name` and `--default-backup-storage-class-name `are abandoned, storage class defaults to Kubernetes default storage class right now. If you have set default storage class different than Kubernetes default storage class, please set them explicitly in your tidb cluster helm or YAML files. ([#1581](https://github.com/pingcap/tidb-operator/pull/1581), [@cofyc](https://github.com/cofyc))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- `--default-storage-class-name` and `--default-backup-storage-class-name `are abandoned, and the storage class defaults to Kubernetes default storage class right now. If you have set default storage class different than Kubernetes default storage class, please set them explicitly in your TiDB cluster helm or YAML files. ([#1581](https://github.com/pingcap/tidb-operator/pull/1581), [@cofyc](https://github.com/cofyc))
```

File: CHANGELOG-1.1.md [hunk 8]
```diff
@@ -1,10 +1,40 @@
...
+
+
+## Other Notable Changes
+
+- Allow users to configure affinity and tolerations for `Backup` and `Restore` ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
+- Make AdvancedStatefulSet and Admission Webhook could work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Allow AdvancedStatefulSet and Admission Webhook to work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
```

File: CHANGELOG-1.1.md [hunk 9]
```diff
@@ -1,10 +1,40 @@
...
+## Changes since v1.1.0-beta.1
+
+## Action Required
+
+- `--default-storage-class-name` and `--default-backup-storage-class-name `are abandoned, storage class defaults to Kubernetes default storage class right now. If you have set default storage class different than Kubernetes default storage class, please set them explicitly in your tidb cluster helm or YAML files. ([#1581](https://github.com/pingcap/tidb-operator/pull/1581), [@cofyc](https://github.com/cofyc))
+
+
+## Other Notable Changes
+
+- Allow users to configure affinity and tolerations for `Backup` and `Restore` ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Allow users to configure affinity and tolerations for `Backup` and `Restore`. ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
```

File: CHANGELOG-1.1.md [hunk 10]
```diff
@@ -1,10 +1,40 @@
...
+- Make AdvancedStatefulSet and Admission Webhook could work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
+- Add a basic deployment example of managing TiDB cluster with custom resources only. ([#1573](https://github.com/pingcap/tidb-operator/pull/1573), [@aylei](https://github.com/aylei))
+- Support TidbCluster Auto-scaling feature based on CPU average utilization load. ([#1731](https://github.com/pingcap/tidb-operator/pull/1731), [@Yisaer](https://github.com/Yisaer))
+- Support user-defined tidb server/client certificate ([#1714](https://github.com/pingcap/tidb-operator/pull/1714), [@weekface](https://github.com/weekface))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Support user-defined TiDB server/client certificate ([#1714](https://github.com/pingcap/tidb-operator/pull/1714), [@weekface](https://github.com/weekface))
```

File: CHANGELOG-1.1.md [hunk 11]
```diff
@@ -1,10 +1,40 @@
...
+- Add an option for tidb-backup chart to allow reusing existing PVC or not for restore ([#1708](https://github.com/pingcap/tidb-operator/pull/1708), [@mightyguava](https://github.com/mightyguava))
+- Add `resources`, `imagePullPolicy` and `nodeSelector` field for tidb-backup chart ([#1705](https://github.com/pingcap/tidb-operator/pull/1705), [@mightyguava](https://github.com/mightyguava))
+- Add more SAN(Subject Alternative Name)s to tidb server certificate ([#1702](https://github.com/pingcap/tidb-operator/pull/1702), [@weekface](https://github.com/weekface))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Add more SANs (Subject Alternative Name) to TiDB server certificate ([#1702](https://github.com/pingcap/tidb-operator/pull/1702), [@weekface](https://github.com/weekface))
```

File: CHANGELOG-1.1.md [hunk 12]
```diff
@@ -1,10 +1,40 @@
...
+- Able to migrate existing Kubernetes StatefulSets to Advanced StatefulSets automatically when AdvancedStatfulSet feature is enabled ([#1580](https://github.com/pingcap/tidb-operator/pull/1580), [@cofyc](https://github.com/cofyc))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Support automatically migrating existing Kubernetes StatefulSets to Advanced StatefulSets when AdvancedStatfulSet feature is enabled ([#1580](https://github.com/pingcap/tidb-operator/pull/1580), [@cofyc](https://github.com/cofyc))
```

File: CHANGELOG-1.1.md [hunk 13]
```diff
@@ -1,10 +1,40 @@
...
+- Fix the bug in admission webhook which causes PD pod deleting error and permit the deleting pod request for pd and tikv when PVC is not found. ([#1568](https://github.com/pingcap/tidb-operator/pull/1568), [@Yisaer](https://github.com/Yisaer))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Fix the bug in admission webhook which causes PD pod deleting error and allow the deleting pod to request for PD and TiKV when PVC is not found. ([#1568](https://github.com/pingcap/tidb-operator/pull/1568), [@Yisaer](https://github.com/Yisaer))
```

File: CHANGELOG-1.1.md [hunk 14]
```diff
@@ -1,10 +1,40 @@
...
+- Limit the restart rate for pd and tikv, only one instance would be restarted at each time ([#1532](https://github.com/pingcap/tidb-operator/pull/1532), [@Yisaer](https://github.com/Yisaer))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Limit the restart rate for PD and TiKV - only one instance would be restarted each time ([#1532](https://github.com/pingcap/tidb-operator/pull/1532), [@Yisaer](https://github.com/Yisaer))
```

File: CHANGELOG-1.1.md [hunk 15]
```diff
@@ -1,10 +1,40 @@
...
+- Adding default ClusterRef namespace for TidbMonitor as same as it deployed and fix the bug when TidbMonitor's Pod can't be created when Spec.PrometheusSpec.logLevel is missing. ([#1500](https://github.com/pingcap/tidb-operator/pull/1500), [@Yisaer](https://github.com/Yisaer))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Add default ClusterRef namespace for TidbMonitor as the same as it is deployed and fix the bug that TidbMonitor's Pod can't be created when Spec.PrometheusSpec.logLevel is missing. ([#1500](https://github.com/pingcap/tidb-operator/pull/1500), [@Yisaer](https://github.com/Yisaer))
```

File: CHANGELOG-1.1.md [hunk 16]
```diff
@@ -1,10 +1,40 @@
...
+- Refine logs for `TidbMonitor` and `TidbInitializer` controller ([#1493](https://github.com/pingcap/tidb-operator/pull/1493), [@aylei](https://github.com/aylei))
+- Avoid unnecessary updates to `Service` and `Deployment` of discovery ([#1499](https://github.com/pingcap/tidb-operator/pull/1499), [@aylei](https://github.com/aylei))
+- Remove some not very useful update events ([#1486](https://github.com/pingcap/tidb-operator/pull/1486), [@weekface](https://github.com/weekface))
```
Comment (yikeke) [inline CHANGELOG-1.1.md]:
```suggestion
- Remove some update events that are not very useful ([#1486](https://github.com/pingcap/tidb-operator/pull/1486), [@weekface](https://github.com/weekface))
```

**Comment by cofyc** [comment]
@yikeke Thanks!

File: charts/tidb-operator/values.yaml [hunk 17]
```diff
@@ -12,7 +12,7 @@ rbac:
 timezone: UTC
 
 # operatorImage is TiDB Operator image
-operatorImage: pingcap/tidb-operator:v1.1.0-beta.1
+operatorImage: pingcap/tidb-operator:v1.1.0-beta.2
 imagePullPolicy: IfNotPresent
 
 # tidbBackupManagerImage is tidb backup manager image
```
Comment (LinuxGit) [inline charts/tidb-operator/values.yaml]: pingcap/tidb-backup-manager should be uncomment and  `pingcap/tidb-backup-manager:v1.1.0-beta.2`?

Comment (onlymellb) [reply charts/tidb-operator/values.yaml]: Agree, as from tidb-operator version 1.1, we should keep the backup-related image versions consistent with the tidb-operator version

Comment (cofyc) [reply charts/tidb-operator/values.yaml]: updated, PTAL

**Pull request merged**